src/main/java/mate/controller/DeleteDriverController.java

```diff
@@ -0,0 +1,22 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet("/drivers/delete")
+public class DeleteDriverController extends HttpServlet {
+    private final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        driverService.delete(Long.valueOf(req.getParameter("id")));
+        resp.sendRedirect(req.getContextPath() + "drivers/drivers");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/drivers");
```
fix the same issue in all places